bindings/python/justfile

```diff
@@ -1,29 +1,26 @@
 set fallback
-VENV := env_var_or_default("VENV", "../../.venv")
+VENV := env_var_or_default("VENV", ".venv")
 VENV_BIN := VENV / "bin"
 
 @venv:
   python3 -c "import virtualenv" || python3 -m pip --quiet install virtualenv
   python3 -m virtualenv {{VENV}} --quiet
-
-## Set up virtual environment and install requirements
-@requirements: venv
-  {{VENV_BIN}}/python -m pip --quiet install --upgrade pip
-  {{VENV_BIN}}/pip --quiet install -r requirements.txt
+  {{VENV_BIN}}/python -m pip install --upgrade pip
```

Comment:
inside of a virutalenv all versions of python are the same thing... if it's python3, then you get both `python` and `python3` 

(the main source of this confusion is that archlinux went ahead and made `/usr/bin/python` python3 very early, and every other distro/system did not, because they had system level python dependencies (whereas arch's system tools are all in C or bash) so while the "standard" is to have `python3` and friends, and `python` within virtualenvs, it's kind of... a mess)